code/game/objects/items/weapons/clown_items.dm

```diff
@@ -39,10 +39,9 @@
 	//So this is a workaround. This also makes more sense from an IC standpoint. ~Carn
 	if(user.client && (target in user.client.screen))
 		to_chat(user, "<span class='notice'>You need to take that [target.name] off before cleaning it.</span>")
-	else if(target == user && user.a_intent == "grab")
+	else if(target == user && user.a_intent == I_GRAB)
 		var/mob/living/carbon/human/muncher = user

```

Comment:
Ok, technically you're right, you don't need the human check as you're using get_species(), however typecasting without checking the type is really not good practice. It would be all to easy for someone else to come to this code, see this var and assume they can use it to access anything a human has access to.